packages/oak-prosemirror-editor/lib/core/Editor/index.styl

```diff
@@ -0,0 +1,89 @@
+.oak-prosemirror.oak-text-editor
+  background: var(--oak-text-editor-color)
+  border-radius: 4px
+  width: 100%
+
+  .oak-text-input
+    padding: 10px
+
+  .oak-toolbar
+    background: var(--oak-text-editor-toolbar-color)
+    border-radius: 4px 4px 0 0
+    padding: 4px 5px
+    display: flex
+    align-items: center
+
+    a.oak-toolbar-button
+      border-radius: 2px
+      display: flex
+      margin-right: 5px
+      align-items: center
+      justify-content: center
+      text-decoration: none
+      padding: 2px
+
+      &.oak-active
+        .oak-icons
+          color: var(--text-color)
+
+      &.oak-underline
+        .oak-icons
+          font-size: 18px
+
+      &.oak-color-button
+        margin-right: 20px
+
+      &.oak-size-button
+        margin-right: 0
+
+        i
+          font-size: 18px
+
+      &.oak-text-left
+        margin-left: 20px
+
+    .oak-icons
+      color: $color-shuttle-gray
+      font-size: 20px
+
+    .oak-text-size
+      font-size: 14px
+      margin: 0 10px
+
+  .oak-text-editable
+    box-sizing: border-box
+    padding: 10px
+    width: 100%
+
+    a
+      color: blue !important
```

Comment:
is important mandatory here?

Reply:
To be honest i've CP this stylesheet from slate addon, and i've not test yet but the important flag does not seems to be mandatory, as the color will be overload by inline `color` css property implied by a new `color` segment. I think I can delete it.